modules/manage/partials/whole-cluster-restore.adoc

```diff
@@ -15,7 +15,7 @@ NOTE: You cannot use whole cluster restore if the target cluster is in xref:{lin
 ====
 Whole cluster restore is not a fully-functional disaster recovery solution. It does not provide snapshot-style consistency. Some partitions in some topics will be more up-to-date than others. Committed transactions are not guaranteed to be atomic.
 
-If you need to restore only a subset of topic data, consider using xref:{link-tiered-storage}#remote-recovery[remote recovery] instead of a whole cluster restore.
+If you need to restore only a subset of topic data, consider using xref:manage:topic-recovery.adoc[topic recovery] instead of a whole cluster restore.
```

Comment:
Line 51 - might be a separate issue, but it looks like the cluster property has moved.

Reply:
@asimms41 ah yes, we moved object storage (tiered storage) properties to a [new page](https://deploy-preview-677--redpanda-docs-preview.netlify.app/current/reference/properties/object-storage-properties/). I'll fix this one but I'm not sure if we have an issue yet to fix the old links from cluster-properties.adoc. I'll ask Paulo.